src/commands/basemaps-github/create-pr.ts

```diff
@@ -61,6 +61,32 @@ async function parseTargetInfo(
   return { name: standardizeLayerName(name), epsg: epsg.code, title, region };
 }
 
+/**
+ * Pass vector target location with the following format
+ * s3://linz-basemaps-staging/vector/3857/53382-nz-roads-addressing/01HSF04SG9M1P3V667A4NZ1MN8/53382-nz-roads-addressing.tar.co
+ * s3://linz-basemaps-staging/vector/3857/topographic/01HSF04SG9M1P3V667A4NZ1MN8/topographic.tar.co
+ */
+async function parseVectorTargetInfo(target: string): Promise<{ name: string; title: string; epsg: EpsgCode }> {
+  logger.info({ target }, 'CreatePR: Get the layer information from target');
+  const url = new URL(target);
```

Comment:
Could we read all this information from the collection.json/item.json rather than trying to parse values from URLs?

Reply:
I had a look of ETL stac files, the only thing I think is useful here is the item.properties['proj:epsg'], So I thought just parse url should be enough for now.

https://github.com/linz/basemaps-team/blob/a791140c3d76ef0a7c4967082de759cb89f51876/packages/ETL/src/stac/stac.ts#L15